content/300-guides/300-upgrade-guides/200-upgrading-versions/600-upgrading-to-prisma-5.mdx

```diff
@@ -0,0 +1,321 @@
+
+#### `OR` clauses
+
+The following code in Prisma 4:
+
+```js
+prisma.user.findMany({
+  OR: { email: 'foo@example.com' },
+})
+```
+
+Will need to be changed to the following in Prisma 5:
+
+```js highlight=2;normal
+prisma.user.findMany({
+  OR: [{ email: 'foo@example.com' }],
+})
+```
+
+`OR` clauses will only accept array values.
+
+#### `in` and `notIn` clauses
+
+Similar to `OR`, `in` and `notIn` require array values.
+
+Prisma 4:
+
+```js
+prisma.user.findMany({
+  id: { in: 123 },
+})
+
+prisma.user.findMany({
+  id: { notIn: 123 },
+})
+```
+
+Prisma 5:
+
+```js highlight=2,6;normal
+prisma.user.findMany({
+  id: { in: [123] },
+})
+
+prisma.user.findMany({
+  id: { notIn: [123] },
+})
+```
+
+If your `in` and `notIn` values are only one element, you can also update your code to not use these clauses:
+
+```js highlight=2,6;normal
+prisma.user.findMany({
+  id: 123,
+})
+
+prisma.user.findMany({
+  id: { not: 123 },
+})
+```
+
+#### `path` argument for filtering on JSON fields in Postgres
+
+[When filtering on JSON fields in a Postgres table](https://www.prisma.io/docs/concepts/components/prisma-client/working-with-fields/working-with-json-fields#filter-on-a-json-field) the `path` argument now only accepts an array.
+
+Prisma 4:
+
+```prisma
+datasource db {
+  provider = "postgresql"
+}
+
+model User {
+  id       String @id
+  settings Json
+}
+```
+
+```js
+prisma.user.findMany({
+  where: {
+    settings: {
+      path: 'someSetting',
+      equals: someValue,
+    },
+  },
+})
+```
+
+Prisma 5
+
+```prisma
+datasource db {
+  provider = "postgresql"
+}
+
+model User {
+  id       String @id
+  settings Json
+}
+```
+
+```js highlight=4;normal
+prisma.user.findMany({
+  where: {
+    settings: {
+      path: ['someSetting'],
+      equals: someValue,
+    },
+  },
+})
+```
+
+<Admonition type="info">
+
+Note: This change only affects PostgreSQL databases. MySQL databases are not affected.
+
+</Admonition>
+
+#### Composite and Scalar lists
+
+Inputs for operations on Composite and Scalar lists now only accept array values. The following examples all use the following schema:
+
+```prisma
+model Post {
+  id String @id @default(uuid())
+  commentsList Comment[]
+  tags String[]
+}
+
+type Comment {
+  text String
+}
+```
+
+##### Composite list filtering
+
+[When filtering records based on a composite list field](https://www.prisma.io/docs/concepts/components/prisma-client/composite-types#finding-records-that-contain-composite-types-with-find-and-findmany) you will only be able to supply array values.
+
+Prisma 4:
+
+```js
+prisma.post.findMany({
+  commentsList: { equals: { text: 'hello' } },
+})
+```
+
+Prisma 5:
+
+```js highlight=2;normal
+prisma.post.findMany({
+  commentsList: { equals: [{ text: 'hello' }] },
+})
+```
+
+##### Composite list input
+
+If you exclude `equals`, you still must supply an array value for composite list fields.
```

Comment:
I don't understand these two headlines here. 

It seems to me both are filtering, just ones using the `equals` explicitly and once using the remaining shortcut that you can leave that off?